modules/apps/transfer/types/packet.go

```diff
@@ -64,3 +67,13 @@ func (ftpd FungibleTokenPacketData) ValidateBasic() error {
 func (ftpd FungibleTokenPacketData) GetBytes() []byte {
 	return sdk.MustSortJSON(mustProtoMarshalJSON(&ftpd))
 }
+
+// GetPacketSender returns the sender address of the packet data.
+//
+// NOTES:
+//   - The sender address is set at the source chain and not validated by
+//     a signature check in IBC.
+//   - sourcePortID is not used in this implementation.
```

Comment:
```suggestion
// NOTE:
//   - The sender address is set by the packet sender and may not have been validated a signature check if the packet sender isn't the transfer module.
//   - The sender address must only be used by modules on the sending chain. 
//   - sourcePortID is not used in this implementation.
```

Reply:
Some nits:

Is it necessary to say `if the packet sender isn't the transfer module`. Because in pfm, the packet sender is still technically the transfer module. Pfm just submits a MsgTransfer on behalf of a user I believe.

Should we use `should` instead of `must` in `The sender address must only be used by modules on the sending chain. `

For now, I'll paste your suggestion.